test/paper-dialog-scrollable.html

```diff
@@ -219,6 +235,18 @@
           });
         });
 
+        test('removes bottom divider if content is removed dynamically', function() {
+          var container = fixture('basic');
+          setTimeout(function() {
+            var scrollable = Polymer.dom(container).querySelector('paper-dialog-scrollable');
+            assert.isTrue(scrollable.classList.contains('scrolled-to-bottom'), 'scrollable has scrolled-to-bottom class');
+            Polymer.dom(scrollable).innerHTML = '<p>dummy content</p>';
```

Comment:
Ugh, tests are failing on 1.x, most likely you need to flush the dom updates with a `Polymer.dom.flush()` after this line, or wait for the mutations to happen with another setTimeout :x